### ISSUE TITLE
Sign out Confirmation

### ISSUE THREAD

**Issue by kasem-sm** [PatilShreyas/NotyKT#297]
As issue #287 suggests for a confirmation alert for deleting notes, I think that confirmation before signing out a user is also necessary.

**Comment by PatilShreyas** [comment]
@kasem-sm yeah, that's nice to have 😃

**Comment by kasem-sm** [comment]
Is there anything remaining in here so I may help?

**Comment by PatilShreyas** [comment]
@kasem-sm this is done. Will close ticket once fix is merged in master and released.
